### PR TITLE
Fix dupe port in cluster_info test

### PR DESCRIPTION
**Pull request by sakridge**
#### Problem

Another test uses 8050, this was 8050 by accident, should be port 0.

#### Summary of Changes

Change port and and remove unintended grow file.

Fixes #

### CODE REVIEW

File: core/src/cluster_info.rs [hunk 1]
```diff
@@ -1805,10 +1805,11 @@ mod tests {
 
     #[test]
     fn new_replicator_external_ip_test() {
-        let ip = IpAddr::V4(Ipv4Addr::from(0));
+        let ip = Ipv4Addr::from(0);
         let node =
-            Node::new_replicator_with_external_ip(&Keypair::new().pubkey(), &socketaddr!(0, 8050));
+            Node::new_replicator_with_external_ip(&Keypair::new().pubkey(), &socketaddr!(ip, 0));
```
Comment (rob-solana) [inline core/src/cluster_info.rs]: the other test that uses 8050 should probably be changed, too...

Comment (sakridge) [reply core/src/cluster_info.rs]: oh, to what? something outside fullnode_port_range?

Comment (rob-solana) [reply core/src/cluster_info.rs]: my thinking is to go bind a random port, then pass that number in.  tests that stake out global ports interfere with each other

Comment (sakridge) [reply core/src/cluster_info.rs]: Ok; take a look.

**Pull request merged**